packages/dds/tree/src/test/domains/json/twitter.ts

```diff
@@ -0,0 +1,631 @@
+                    expanded_url: "http://astore.amazon.co.jp/furniturewood-22",
+                    display_url: "astore.amazon.co.jp/furniturewood-…",
+                    indices: [
+                        random.integer(0, 199),
+                        random.integer(0, 199),
+                    ],
+                },
+            ],
+        };
+    }
+    if (shouldAddDescriptionUrlsEntity) {
+        user.entities.description.urls.push(
+            {
+                url: "http://t.co/8E91tqoeKX",
+                expanded_url: "http://ameblo.jp/2no38mae/",
+                display_url: "ameblo.jp/2no38mae/",
+                indices: [
+                    random.integer(0, 199),
+                    random.integer(0, 199),
+                ],
+            },
+        );
+    }
+
+    return user;
+}
+/* eslint-enable */
+
+function getBasicJapaneseAlphabetString() {
+    // Japanese Hiragana
+    return createAlphabetFromUnicodeRange(0x3041, 0x3096) +
+        // Japanese Katakana (Full Width)
+        createAlphabetFromUnicodeRange(0x30A0, 0x30FF) +
+        // Japanese Kanji Alphabet (CJK Unified Ideographs)
+        createAlphabetFromUnicodeRange(0x3400, 0x4DB5) +
+        createAlphabetFromUnicodeRange(0x4E00, 0x9FCB) +
+        createAlphabetFromUnicodeRange(0xF900, 0xFA6A);
+}
+
+// This is specifically formatted like the twitter json dates
+// (<3-letter-weekday> MMM DD HH:MM:SS <4-digit-TimezoneOffset> YYYY)
+function getRandomDateString(random = makeRandom(), start: Date, end: Date) {
+    const dateS = new Date(random.integer(+start, +end)).toString();
+    return `${dateS.substring(0, 10)} ${dateS.substring(16, 24)} ` +
+        `${dateS.substring(28, 33)} ${dateS.substring(11, 15)}`;
+}
+
+// Source for unicode ranges:
+// https://stackoverflow.com/questions/19899554/
+// unicode-range-for-japanese#:~:text=To%20summarize%20the%20ranges%3A,Katakana%20(%2030a0%20%2D%2030ff)
+// or more direct source:
+// http://www.localizingjapan.com/blog/2012/01/20/regular-expressions-for-japanese-text/
+export function isJapanese(ch: string) {
+    // Japanese Hiragana
+    return (ch >= "\u3041" && ch <= "\u3096"
+        // Japanese Katakana (Full Width)
+        || ch >= "\u30A0" && ch <= "\u30FF"
+        // Japanese Kanji Alphabet (CJK Unified Ideographs)
+        || ch >= "\u3400" && ch <= "\u4DB5"
+        || ch >= "\u4E00" && ch <= "\u9FCB"
+        || ch >= "\uF900" && ch <= "\uFA6A"
+        // Kanji Radicals
+        || ch >= "\u2E80" && ch <= "\u2FD5"
+        // Katakana and Punctuation (Half Width)
+        || ch >= "\uFF5F" && ch <= "\uFF9F"
+        // Miscellaneous Japanese Symbols and Characters
+        || ch >= "\u31F0" && ch <= "\u31FF"
+        || ch >= "\u3220" && ch <= "\u3243"
+        || ch >= "\u3280" && ch <= "\u337F");
+}
+
+export function isAlphaLatin(ch: string) {
+    // range 1: ABCDEFGHIJKLMNO0050PQRSTUVWXYZ
+    return (ch >= "\u0041" && ch <= "\u005A")
```

Comment:
Nit: '0050' in the comment above looks like an accidental paste?

Reply:
Yep. Nice catch, fixed.